About/former.md

```diff
@@ -0,0 +1,26 @@
+
+
+# Former Board Members:
+
+
+
+## Jono Bacon
+
+![Jono Bacon](https://github.com/chaoss/website/blob/master/About/images/jono.jpg)

```

Comment:
We need to use the github.io URL that is generated for these pictures. See example: https://raw.githubusercontent.com/chaoss/website/master/CHAOSScon/2020EU/speakers.md